packages/e2eTesting/integration/ActionableItem.spec.ts

```diff
@@ -0,0 +1,44 @@
+context('Actions', () => {
+    before(() => {
+      cy.visit('localhost:8080/#/components/ActionableItem')
+    })
+    it('Validate Actionnable item without onItemClick Method',() => {
+      cy.get('div[class*="actionable-item-content"]')
```

Comment:
I don't know enough about Cypress but would it make sense putting repeated paths into variables?

like `const actionableItemContent = 'div[class*="actionable-item-content"]'`?

Reply:
Got it! :)